install/relay-credentials.sh

```diff
@@ -15,9 +16,26 @@ if [[ ! -f "$RELAY_CREDENTIALS_JSON" ]]; then
 
   $dcr \
     --no-deps \
-    --volume "$(pwd)/$RELAY_CONFIG_YML:/tmp/config.yml" \
-    relay --config /tmp credentials generate --stdout \
-    > "$RELAY_CREDENTIALS_JSON"
+    --volume "$(pwd)/$RELAY_DIRECTORY:/tmp/relay" \
+    --entrypoint \"/bin/bash\" relay -c \
+    "chown -R 10001 /tmp/relay"
+
+  ls -l $RELAY_DIRECTORY
+
+  $dcr \
+    --no-deps \
+    --volume "$(pwd)/$RELAY_DIRECTORY:/tmp/relay" \
+    relay --config /tmp/relay credentials generate
```

Comment:
I'd say, in the final version, we can just use the mounter `relay` config dir for this?

Reply:
Can you elaborate please?

Reply:
We already mount the `relay` directory for the `relay` service and set that as the config folder so you should not need the `--volume "$(pwd)/$RELAY_DIRECTORY:/tmp/relay"` part and `--config /tmp/relay` part.

Reply:
We have this:
https://github.com/getsentry/self-hosted/blob/da8f4903e4cfe47ca4fa5bf68265b62f795c60a0/docker-compose.yml#L340-L358
which mounts relay directory read-only and AFAICS there is no `--config`.

Reply:
We _can_ mount it as writable for that run but yeah, I get your point :)